src/tables/editable-table.js

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Table,
+  TableRow,
+  TableHeader,
+  TableHeaderColumn,
+  TableRowColumn,
+  TableBody } from 'material-ui/Table';
+import Toggle from 'material-ui/Toggle';
+import ActionButton from '../action-button';
+import FlatButton from 'material-ui/FlatButton';
+import TableRowHeader from './table-row-header';
+import * as Colors from 'material-ui/styles/colors';
+import { split, filter } from 'lodash';
+
+export default class EditTable extends Component {
+  constructor(props) {
+    super(props);
+    this.renderRowColumns = this.renderRowColumns.bind(this);
+    this.renderRow = this.renderRow.bind(this);
+    this.renderHeaders = this.renderHeaders.bind(this);
+  }
+
+  renderRowColumns(row, fields) {
+    let index = 0;
+    return fields.map((field) => {
+      const tokens = split(field.val, '.', 3);
+      let value = row[tokens[0]];
+      index += 1;
+      for (let i = 1; i < tokens.length; i++) {
+        value = value ? value[tokens[i]] : '';
+      }
+      if (field.edit) {
```

Comment:
You need to test if the field is of type boolean before replacing it with toggle.